ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -429,6 +444,28 @@ export const WorkspaceAbout = fp.flow(
                   </div>
                 )}
             </div>
+
+            <div>
+              <h3 style={{ marginBottom: '0.5rem' }}>Billing</h3>
+              <StyledExternalLink
```

Comment:
Please add the TooltipTrigger to explain why it is disabled, per https://github.com/all-of-us/workbench/blob/main/ui/src/app/pages/admin/admin-workspace.tsx#L152-L163